content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,17 @@
+## Bug fixes and minor enhancements
+
+- Add a precise SHA reference to the release manifest ([Pull Request 14959](https://github.com/istio/istio/pull/14959))
+- Fix regression caused by forward port merge error ([Issue 14976](https://github.com/istio/istio/issues/14976))
```

Comment:
Maybe "fix duplicate CRD being generated in the install"

Reply:
```suggestion
- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
```